extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -694,4 +726,99 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 }
 
+/**
+ * Keeps track of the third-party vendors' cross-domain iframes. Only one
+ * static instance will be created, which will keep and manage ALL amp-analytics
+ * tags.
+ */
+class AmpAnalyticsFrames {
```

Comment:
This likely belongs in another class, hopefully one that reduces the duplication between this and `transport.js`.